docs/templates/index.html

```diff
@@ -59,6 +59,16 @@ <h1 id="nav-title">Documentation</h1>
     if you're interested in helping out. </p>
 
 
+<div id="info" class="grid-container padding">
+    <div class="grid-item">
+    </div>
+    <div class="grid-item">
+    </div>
+    <div class="grid-item">
+        <p class="box-title">Next step</p>
+        <p class="box placeholder"><a class="link" href="getting-started.html">Getting started</a></p>
+    </div>
+</div>
```

Comment:
Let's make this take the full width instead
```suggestion
 <p class="box-title">Next step</p>
 <p class="box placeholder">Visit the <a class="link" href="getting-started.html">Getting started</a> page to learn more on how to install Skript and create your first script!</p>
```

Reply:
Okay, let me check this first. I'll compare the results.

Reply:
Looks better. Done.